x-pack/plugins/ml/kibana.jsonc

```diff
@@ -31,7 +31,8 @@
       "unifiedSearch",
       "savedObjectsFinder",
       "savedObjectsManagement",
```

Comment:
`savedObjectsManagement` is only needed so we can pass it to `SavedObjectFinder` from `@kbn/saved-objects-finder-plugin/public`
@majagrubic you look to be the last person who touched code in `SavedObjectFinder`, I don't suppose you know if there are plans to remove the dependency on `savedObjectsManagement`?

We use `SavedObjectFinder` [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/ml/public/application/jobs/new_job/pages/components/datafeed_step/components/data_view/change_data_view.tsx#L151) and [here](https://github.com/elastic/kibana/blob/536a8d4194b695249aa85387267ead62e7e7f862/x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/source_selection/source_selection.tsx#LL120C4-L120C4)